app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java

```diff
@@ -43,39 +45,48 @@ public void tearDown() {
     }
 
     @Test
-    public void showSettingsViews() throws InterruptedException {
-        onView(withId(R.id.navUrlInput)).check(matches(isDisplayed()));
-
-        // current settings list view
-        onView(withId(R.id.container_web_render)).check(matches(isDisplayed()));
-
+    public void showSettingsUsageData() throws InterruptedException {
         linearNavigateToSettingsChannel();
 
-        onView(withId(R.id.settings_tile_telemetry)).check(matches(isDisplayed()));
```

Comment:
Should we move this `onView` check lower, to verify we're clicking into the Telemetry tile? (in case the tile order changes)

Reply:
Since there is another test that checks the layout of the UI test, and this test only collects screenshots, I figured I could remove all the element checks to simplify it. (we manually check screenshots to make sure they match the title) We could click the DOM with the right tile ID instead of using `pressDPadCenter()` if we want to be sure.  Should I change it to work that way?